types/three/examples/jsm/loaders/GLTFLoader.d.ts

```diff
@@ -65,7 +65,7 @@ export class GLTFLoader extends Loader {
         onError?: (event: ErrorEvent) => void,
     ): void;
```

Comment:
Why doesn't `parse` return a GLTF?

Reply:
It's not implemented that way https://github.com/mrdoob/three.js/blob/00d3363e2d8a6d43dfdb75e50e3ff553b3c2f932/examples/jsm/loaders/GLTFLoader.js#L278

The semantics of `parse` vs `parseAsync` in Three.js loaders is not "synchronous versus asynchronous", but "asynchronous using callbacks versus asynchronous using Promises".

Reply:
The promise parseAsync returns resolves to a GLTF object